packages/components/organisms/f-header/src/assets/scss/nav.scss

```diff
@@ -0,0 +1,123 @@
+@import './common.scss';
+
+$nav-transition-duration : 250ms;
+
+$nav-trigger-width : 56px;
+$nav-trigger-height : 48px;
+$nav-trigger-focus-color : $blue;
+$nav-trigger-focus-bg : $blue--offWhite;
+
+$nav-popover-transition-delay : 200ms;
+$nav-popover-transition-duration : 200ms;
+
+$countrySelector-flag-width : 16px;
+$countrySelector-flag-height : 16px;
+
+@mixin nav-container-visible () {
+    overflow-y: auto;
+    left: 0;
+    opacity: 1;
+    z-index: zIndex(high);
+    transition: opacity $nav-transition-duration ease-in-out,
+        z-index 0s linear;
+}
+
+.c-nav-list {
+    position: relative;
+}
+
+.c-nav-list,
+.c-nav-popoverList {
+    margin-top: 0;
+    margin-left: 0;
+    list-style: none;
+    list-style-image: none;
+    padding: 0;
+
+    &>li {
+        margin-bottom: 0;
+
+        &:before {
+            content: none;
+        }
+    }
+
+    @include media('<=mid') {
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.c-nav-popoverList--twoColumns {
+    @include media('>mid') {
+        column-count: 2;
+    }
+}
+
+// Navigation Trigger
+// This is the checkbox that controls the menu active state without JS via :checked
+.c-nav-trigger {
+    position: absolute;
+    width: $nav-trigger-width;
+    height: $nav-trigger-height;
+    top: -100px;
+    left: -100px;
+
+    @include media('>mid') {
+        display: none;
+    }
+
+    &:checked~.c-nav-container {
```

Comment:
There are some failing lint errors in the build mostly around spacing between selectors i.e. here there should be spaces around the `~`.